x-pack/plugins/observability_solution/slo/public/hooks/use_create_burn_rate_rule.tsx

```diff
@@ -40,6 +67,9 @@ export function useCreateRule<Params extends RuleTypeParams = never>() {
     },
     {
       onError: (_err) => {
+        if (loadingToastId) {
```

Comment:
There is a `onSettled` callback that is called after success or error, we could use that instead of duplicating the logic in both code in onError/onSuccess.